frontend/components/ExportBanner.js

```diff
@@ -29,20 +31,44 @@ const Square = ({ fill }) => html`
 //@ts-ignore
 window.enable_secret_pluto_recording = true
 
+const getId = (/** @type {string} */ url) => {
+    const id = url.split("id=")[1]
+    return id
+}
+
 export const ExportBanner = ({ onClose, notebookfile_url, notebookexport_url, start_recording }) => {
+    const { isDesktop, conditionalCallback } = usePlutoDesktop(undefined)
+
+    const id = getId(notebookfile_url)
+
+    const exportNotebook = (/** @type {{ preventDefault: () => void; }} */ e, /** @type {0 | 1 | 2 | 3} @note see *types.d.ts* for more info.*/ type) => {
+        e.preventDefault()
```

Comment:
(Don't `preventDefault` if you are not in desktop mode, or the export in regular Pluto won't work anymore)

Reply:
sorted